config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  root 'homes#index'
+  root 'podcasts#index'
+  resources :podcasts, only: [:index]
```

Comment:
Do you definitely want to be able to access this page at both URLs, or just one or the other?

If one or the other, just include one of these lines.

Another optional refactor on line 3: remove the array brackets as so:
```
resources :podcasts, only: :index
```